### PR TITLE
General brush up of PowerToys docs

### PR DESCRIPTION
**Pull request by Jay-o-Way**
- occasional spelling fixes
- use "select" for interaction, and for [checkbox state](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark)
- use "screen", "display" according to [Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/screen)
- "tab" for a page in the Settings window
- format of "arrow keys"
- and such...

TO-DO

- [ ] add new plugin: Value generator
- [x] add new setting for highlighter

### CODE REVIEW

File: hub/powertoys/color-picker.md
Comment (alvinashcraft) [inline hub/powertoys/color-picker.md]:
```suggestion
The editor lets you see the history of (up to 20) picked colors and copy their representation in any predefined string format. In PowerToys Settings, you can configure which color formats are visible in the editor and in what order that they appear.
```

File: hub/apps/whats-new/latest-doc-updates.md
Comment (alvinashcraft) [inline hub/apps/whats-new/latest-doc-updates.md:33]: Is this view parameter needed? See build suggestion. It should either be removed or add `&preserve-view=true` to the URL if it's needed.

File: hub/apps/whats-new/latest-doc-updates.md
Comment (alvinashcraft) [inline hub/apps/whats-new/latest-doc-updates.md:67]: Is this view parameter needed on line 68? See build suggestion. It should either be removed or add `&preserve-view=true` to the URL if it's needed.

File: hub/powertoys/image-resizer.md
Comment (alvinashcraft) [inline hub/powertoys/image-resizer.md]:
```suggestion
The fallback encoder is used when the file cannot be saved in its original format. For example, the Windows Meta File (.wmf) image format has a decoder to read the image, but no encoder to write a new image. In this case, the image cannot be saved in its original format. Specify the format the fallback encoder will use: PNG, JPEG, TIFF, BMP, GIF, or WMPhoto settings. **This is not a file type conversion tool, but only works as a fallback for unsupported file formats.**
```
"Specify which format" would also be OK.

File: hub/powertoys/quick-accent.md
Comment (alvinashcraft) [inline hub/powertoys/quick-accent.md:71]: Do you want to leave the Description blank for this table entry?

File: hub/powertoys/video-conference-mute.md
Comment (alvinashcraft) [inline hub/powertoys/video-conference-mute.md:47]: Is there some description that can be provided for this entry?

**Review by alvinashcraft — COMMENTED** [review]
Looks much better, overall. I left some minor comments and questions.

File: hub/apps/whats-new/latest-doc-updates.md
Comment (Jay-o-Way) [reply hub/apps/whats-new/latest-doc-updates.md:33]: Uhm, honestly I have no idea, haha. I just let VS Code (Markdown plug-in) do a quick "Format Document" 😅
#wasntme

File: hub/apps/whats-new/latest-doc-updates.md
Comment (Jay-o-Way) [reply hub/apps/whats-new/latest-doc-updates.md:67]: Uhm, honestly I have no idea, haha. I just let VS Code (Markdown plug-in) do a quick "Format Document" 😅

File: hub/powertoys/quick-accent.md
Comment (Jay-o-Way) [reply hub/powertoys/quick-accent.md:71]: Yeah, when the description is the same text as the setting, it's not needed

File: hub/powertoys/video-conference-mute.md
Comment (Jay-o-Way) [reply hub/powertoys/video-conference-mute.md:47]: I think the setting is pretty self-explanatory.

File: hub/apps/whats-new/latest-doc-updates.md
Comment (alvinashcraft) [reply hub/apps/whats-new/latest-doc-updates.md:33]: OK. Probably fine to leave it for now since it's not something you added.

File: hub/apps/whats-new/latest-doc-updates.md
Comment (alvinashcraft) [reply hub/apps/whats-new/latest-doc-updates.md:67]: OK. Probably fine to leave it for now since it's not something you added.

**Review by alvinashcraft — APPROVED** [review]
Looks good to me now. Do you want Clint or anyone from the team to take a look at it before I merge?

**Comment by Jay-o-Way** [comment]
> Looks good to me now. Do you want Clint or anyone from the team to take a look at it before I merge?

Thank you. It would be a courtesy to give Clint a chance, yes. Or else i'll leave it up to you.

**Comment by alvinashcraft** [comment]
Sounds good. I'll hold off until tomorrow so folks have a chance. Thanks for the contribution!

File: hub/powertoys/always-on-top.md
Comment (NathanD-GDS) [inline hub/powertoys/always-on-top.md]:
```suggestion
From the Settings tab, you can configure the following options:
```

We should [use the active voice](https://learn.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice) in documentation where possible.

File: hub/powertoys/awake.md
Comment (NathanD-GDS) [inline hub/powertoys/awake.md:33]: This might not be in the scope of this PR, but [UI text like setting names should be in bold](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions#in-documentation-and-technical-content). If it's out of scope for this, I might make a pass of the docs soon because it's quite inconsistent across PowerToys.

Comment (NathanD-GDS) [reply hub/powertoys/awake.md:33]: Just seen you've added bold to other UI text on this page - nice!

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
To start using Color Picker, enable it in the PowerToys Settings (**Color Picker** tab).
```

Tab names should be in bold.

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
### Enabling Color Picker
```

Again, this could be out of scope for the PR, but I think the headings could do with being a bit more specific on this page.

We should also make them more consistent - if it's 'Getting' in the H2, it should be 'Enabling' and 'Activating' in the H3s.

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
### Activating Color Picker
```

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
### Selecting colors
```

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
## Using the Color Picker editor
```

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
- **Activation shortcut**
- **Activation behaviour**
- **Default color format**, such as HEX, RGB, and more.
```

I think these should be the exact UI text and, therefore, in bold.

Again, let me know if this is out of scope for this PR!

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
- **Show color name** shows a high-level representation of the color. For example: Light Green, Green, Dark Green.
``` 

See comment on lines 65-67

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [reply hub/powertoys/color-picker.md]: If this is out of scope, it might be worth impoving the lead-in line on line 63 to something like:

> In Color Picker, you can configure:

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
To disable a color format, turn off the toggle next to that color format. Disabled color formats are not deleted but are not shown in the Color Picker editor window.
```

Change the way that [toggles are described to be in line with the style guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui).

Removed the capital letter from 'Editor' to be consistent with the editor section earlier on the page.

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
To edit a color format, select it from the list. You can edit the color format's **Name** and **Format** in the dialog.
```

File: hub/powertoys/color-picker.md
Comment (NathanD-GDS) [inline hub/powertoys/color-picker.md]:
```suggestion
You can add, edit, and delete color formats. The syntax is described in the color format editor dialog.
```

Replaces passive voice with active voice.

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
FancyZones is a window manager utility for arranging and snapping windows into efficient layouts to improve your workflow and restore layouts quickly. You can define a set of zone locations to use as targets for windows on your desktop. When you drag a window into a zone, or enter the associated keyboard shortcut, the window is resized and repositioned to fill that zone.
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
Drag the window. By default, you'll also need to select and hold the <kbd>Shift</kbd> key. You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone.
```

I don't think you need to specify that the activation key can be disabled at this point because it is implied with 'By default'.

'Select' is preferred to 'press' when [describing keyboards](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts).

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
If both **Hold Shift key to activate zones while dragging** and **Use non-primary mouse button to toggle zone activation** are cleared, zones will appear immediately after you start dragging the window.
```

Bold setting names

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
Select **Override Windows Snap** in the FancyZones settings. Use <kbd>Win</kbd>+[arrow keys] to snap a window to a zone. Use **Move windows based on** to choose whether to move windows based the zone index or a window's relative position.
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
If two zones are adjacent, you can snap a window to the sum of their area (rounded to the minimum rectangle that contains both). When the mouse cursor is near the common edge of two zones, both zones are activated simultaneously, allowing you to drop the window into both zones.
```

Passive voice

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
### Snapping to multiple zones with the mouse and keyboard
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
Select **Override Windows Snap** in the FancyZones settings and select **Move windows based on: Relative position**. Use <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+[arrow keys] to expand the window to multiple zones.
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
To snap applications that are elevated (such as Windows Terminal or Task Manager), run PowerToys in administrator mode. Read [Running as administrator](administrator.md) for more information.
```

Missing comma

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
Launch the zones editor by selecting **Launch layout editor** in the PowerToys Settings tab or by pressing <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>`</kbd> ("back-tick" or "accent grave"). You can change the FancyZones layout editor shortcut in PowerToys Settings.
```

Simplified sentence structure.

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
When you first launch the zones editor, you'll see a list of layouts that can be adjusted by how many windows are on the monitor. Selecting a layout shows a preview of that layout on the screen. The selected layout is applied automatically. Double-clicking a layout will apply it and close the editor.
```

Simplified

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
**Show space around zones** sets the size of margin around each FancyZone window. Enter a custom value for the width of the margin in **Space around zones**. With the zones editor open, change **Show space around zones** after changing the values to see the new value applied.
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
**Distance to highlight adjacent zones** sets a custom value for the amount of space between zones until they snap together, or before both are highlighted enabling them to merge together.
```

File: hub/powertoys/fancyzones.md
Comment (NathanD-GDS) [inline hub/powertoys/fancyzones.md]:
```suggestion
**Default layout for horizontal monitor orientation** and **Default layout for vertical monitor orientation** set which layout to use as the default when the display configuration is changed in the system (for example, if you add a new display).
```

Simplified and removed the `e.g.` abbreviation.

**Review by NathanD-GDS — COMMENTED** [review]
I haven't had time to look at the full PR, but I've left a few suggestions on a few pages.

Let me know if any of them are out of scope for this PR and I'll make a note to action them myself (one day!).

**Comment by Jay-o-Way** [comment]
"a few"? 🤣 LOL nah you've got some good points. I'll process them when I'm back on my computer 👍🏻

File: hub/powertoys/awake.md
Comment (Jay-o-Way) [reply hub/powertoys/awake.md:33]: I think - strictly speaking, you would be correct, but... It's a table full of settings, so type of UI is implied, and honestly I think it's just pretty ugly :)

File: hub/powertoys/color-picker.md
Comment (Jay-o-Way) [reply hub/powertoys/color-picker.md]: Yeah I see the point. My English is pretty good, but the choice between the "Enable" versus "Enabling" form is too ambiguous for me. I'll trust in your suggestions here.

**Comment by Jay-o-Way** [comment]
@crutkas thoughts?

**Review by alvinashcraft — APPROVED** [review]
This looks good to me now. I think we're good to move forward with merging the PR. We can circle back with another PR later to address additional feedback, if necessary.

**Comment by Jay-o-Way** [comment]
PS those build updates are just annoying as 💩
😆 Sorry, just had to say it. At least it would be nice if all the older comments would be removed upon a new comment...

**Pull request merged**
